src/runtime/internal/dom.ts

```diff
@@ -90,10 +90,12 @@ export function attr(node: Element, attribute: string, value?: string) {
 }
 
 export function set_attributes(node: Element & ElementCSSInlineStyle, attributes: { [x: string]: string }) {
+	//@ts-ignore
```

Comment:
Extreme nit zone: Every other place where we have `@ts-ignore`, there's a space after the `//`.